frontends/ast/simplify.cc

```diff
@@ -156,6 +156,9 @@ Fmt AstNode::processFormat(int stage, bool sformat_like, int default_base, size_
 			arg.type = VerilogFmtArg::INTEGER;
 			arg.sig = node_arg->bitsAsConst();
 			arg.signed_ = node_arg->is_signed;
+		} else if (!required) {
+			log_file_info(filename, location.first_line, "Skipping system task `%s' with non-constant argument at position %zu.\n", str.c_str(), index + 1);
```

Comment:
On second thought, I'm not sure we should just skip things we can't evaluate. If the designer doesn't want it to be evaluated they have `ifndef SYNTHESIS`, no?